cbbpoll/models.py

```diff
@@ -353,3 +353,38 @@ def __repr__(self):
 
     def __str__(self):
         return self.text
+
+
+class Conference(db.Model):
+    __tablename__  = 'conference'
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(160))
+    year = db.Column(db.Integer)
+
+
+class Game(db.Model):
+    __tablename__ = 'game'
+    id = db.Column(db.Integer, primary_key=True)
+    conference_id = db.Column(db.Integer, db.ForeignKey('conference.id'))
+    point_value = db.Column(db.Integer)
+    home_team_id = db.Column(db.Integer, db.ForeignKey('team.id'), nullable=True)
+    away_team_id = db.Column(db.Integer, db.ForeignKey('team.id'), nullable=True)
+    next_game = db.Column(db.Integer, db.ForeignKey('game.id'), nullable=True)
+    winner_is_home = db.Column(db.Boolean, nullable=True)
```

Comment:
It should be possible to define a table-level `UNIQUE` constraint on the combination of `next_game_id` and `winner_is_home`